### PR TITLE
How to test Blazor components in ABP.

### LINKED ISSUE #10368: How to test Blazor components in ABP.
https://support.abp.io/QA/Questions/1979

### PR DESCRIPTION
**Pull request by maliming**
Resolve #10368

### CODE REVIEW

**Comment by hikalkan** [comment]
@enisn can you please test and comment on that? @braim23 you can check the text.

File: docs/en/Community-Articles/2022-01-10-How-to-test-Blazor-components-in-ABP/POST.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+## Source Code
+
+You can find the source of the example solution used in this article [here](https://github.com/abpframework/abp-samples/tree/master/BlazorPageUniTest).
+
+
+In this article I will use [bUnit](https://github.com/bUnit-dev/bUnit) for a simple test of the Blazor component.
+
+## Getting Started
+
+Use ABP CLI to create a blazor app, then add `BookStore.Blazor.Tests` unit test project to the solution.
```
Comment (enisn) [inline docs/en/Community-Articles/2022-01-10-How-to-test-Blazor-components-in-ABP/POST.md]: I think making it ordered is better.

> - Use ABP CLI to create a blazor app:
> `abp new BookStore -t app -u blazor`
> - Then add `BookStore.Blazor.Tests` unit test project to the solution.

**Comment by enisn** [comment]
It works well.

But should we share full content of `BookStore.Blazor.Tests.csproj` file? 
Because I've created a new project with `dotnet new classlib` command and tests couldn't be executed until I add `Microsoft.NET.Test.Sdk` reference

```xml
  <ItemGroup>
    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.11.0" />
  </ItemGroup>
```

Or we can just specify creating a new project with the following command: `dotnet new xunit -n BookStore.Blazor.Tests `

**Comment by maliming** [comment]
I Updated some content.

**Review by enisn — APPROVED** [review]
Awesome🎉
Completely clear and understandable 👍

**Comment by braim23** [comment]
Hello sir, I've seen you use the articles "the", "a", and "an" sometimes and other times not. If you think I made a mistake in some and you're quite sure about your context then you can ignore whichever you like, otherwise I think it looks perfect like this :)

**Review by enisn — APPROVED** [review]
Thanks for the grammatical changes @braim23 :) 👍 

> Hello sir, I've seen you use the articles "the", "a", and "an" sometimes and other times not. If you think I made a mistake in some and you're quite sure about your context then you can ignore whichever you like, otherwise I think it looks perfect like this :)

You may need to check @maliming

**Review by braim23 — APPROVED** [review]
And so sorry for answering this late, trust me I'm faster than this, it's just these couple days..

**Pull request merged**